test/modules/MetricsValidator/src/tests/ValidateDocumentedMetrics.cs

```diff
@@ -106,11 +128,15 @@ protected override async Task Test(CancellationToken cancellationToken)
 
         async Task SeedMetrics(CancellationToken cancellationToken)
         {
+            string deviceId = Environment.GetEnvironmentVariable("IOTEDGE_DEVICEID");
+
             await this.moduleClient.SendEventAsync(new Message(Encoding.UTF8.GetBytes("Test message to seed metrics")), cancellationToken);
 
             const string methodName = "FakeDirectMethod";
             await this.moduleClient.SetMethodHandlerAsync(methodName, (_, __) => Task.FromResult(new MethodResponse(200)), null);
-            await this.moduleClient.InvokeMethodAsync(Environment.GetEnvironmentVariable("IOTEDGE_DEVICEID"), Environment.GetEnvironmentVariable("IOTEDGE_MODULEID"), new MethodRequest(methodName), cancellationToken);
+            await this.moduleClient.InvokeMethodAsync(deviceId, Environment.GetEnvironmentVariable("IOTEDGE_MODULEID"), new MethodRequest(methodName), cancellationToken);
+            // BEARWASHERE -- Lets see if you can ping a M2M dm to edgeAgent via edgeHub
+            await this.moduleClient.InvokeMethodAsync(deviceId, "$edgeAgent", new MethodRequest("ping"), cancellationToken);
```

Comment:
This should work...did it? You could also do it from the test rather than the metricsValidator module; see my earlier comment.

Regardless, please remove the BEARWASHERE comment...

Reply:
Nope. This doesn't work.